### ISSUE TITLE
Hope developers can add Chinese language to the program

### ISSUE THREAD

**Issue by hongruilin** [reorproject/reor#232]
Hope developers can add Chinese language to the program

**Comment by samlhuillier** [comment]
Would you be interested in helping us do this?

**Comment by hongruilin** [comment]
> Would you be interested in helping us do this?

Of course, it's an honor. What should I do? Does the program have a file specifically for storing language texts? Can I translate the text inside into Chinese
